cloud/alerts-notifications/notif-configuration.md

```diff
@@ -3,25 +3,47 @@ id: notif-configuration
 title: Configuring notifications
 ---
 
-By default, notifications are active for all hosts and services. In Centreon Cloud, notifications are sent by the central server.
+By default, no notifications are sent by Centreon Cloud. To activate notification emails for specific resources, you need to create notification rules.
 
-## In which cases are notifications sent?
+Notifications are sent from the central server, according to the central server's time zone.
```

Comment:
This is confusing. Plus, if user uses "Date" macro in message, this would be poller timezone.
I suggest we remove it